runtime/browser/runtime_geolocation_permission_context.h

```diff
@@ -26,7 +26,7 @@ class RuntimeGeolocationPermissionContext
   virtual void RequestGeolocationPermission(
       content::WebContents* web_contents,
       const GURL& requesting_frame,
-      base::Callback<void(content::PermissionStatus)> result_callback);
+      base::Callback<void(bool)> result_callback);
```

Comment:
I think it is. If you look at InvokeGeolocationCallback, https://code.google.com/p/chromium/codesearch#chromium/src/android_webview/native/aw_contents.cc&q=aw_conte&sq=package:chromium&l=1
then they haven't changed java equivalent from bool to enum. That's why I had to change it here. This is in fact similar to the upstream change.